zoomus/util.py

```diff
@@ -145,6 +156,30 @@ def delete_request(
             cookies=cookies,
             timeout=self.timeout)
 
+    def put_request(
```

Comment:
Is there a reason this was placed here instead of in `BaseComponent` like the `post_request` method?

Reply:
I just looked at my code again, and realized that the one in `BaseComponent` was calling the one in `ApiClient`. Looks like this is in the right place =)